libs/git_wrapper/Git_project.ml

```diff
@@ -14,6 +14,9 @@ let force_project_root ?(project_root : Rfpath.t option) (path : Fpath.t) =
     | None -> Rfpath.getcwd ()
     | Some x -> x
   in
+  Logs.debug (fun m ->
+      m "project_root=%s path=%s" (Rfpath.show project_root)
+        (Fpath.to_string path));
```

Comment:
You can open Fpath_.Operators and then use !!path instead of the longer (Fpath.to_string path).